cli/tests/init_module_src_path.sh

```
@@ -0,0 +1,29 @@
+if [ -d "output/init-module" ]; then
+    rm -rf output/init-module
+fi
+
+mkdir -p output/init-module
+cd output/init-module
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)